reprospect/test/sass/instruction/operand.py

```diff
@@ -17,3 +22,42 @@ class MathModifier(StrEnum):
     NEG = '-'
     INV = '~'
     ABS = '|'
+
+class Operand:
+    """
+    Operand patterns.
+    """
+    OPERAND: typing.Final[str] = r'[\w@!\.\[\]\+\-\|~\s]+'
```

Comment:
```suggestion
    #: Match any operand.
    OPERAND: typing.Final[str] = r'[\w@!\.\[\]\+\-\|~\s]+'
```

Reply:
Yeah, I agree, we haven't seen the `@` in operands so far.